sympy/physics/vector/frame.py

```diff
@@ -1,6 +1,7 @@
-from sympy.core.backend import (diff, expand, sin, cos, sympify, eye, symbols,
+from sympy.core.backend import (diff, expand, sin, cos, sympify, eye,
                                 ImmutableMatrix as Matrix, MatrixBase)
-from sympy import (trigsimp, solve, Symbol, Dummy)
+from sympy.core.sympify import sympify as sympify2
```

Comment:
I have found that `core.backend.sympify` and `core.sympify.sympify` have different behaviours to symengine. While the former changes to `symengine.lib.symengine_wrapper.sympify`, the latter remains a sympy function. This has the effect that the former starts converting its inputs to symengine objects while the latter works as usual converting its inputs to sympy objects. So, I had to import the latter as a separate function.